### ISSUE TITLE
Renal vein thrombosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7374]
**Preferred term label:**
Renal vein thrombosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
The presence of a thrombus (blood clot) in the major renal veins or its tributaries, generally leading to passive congestion that develops causes the affected kidney to swell and become engorged, leading to degeneration of nephrons and causing symptoms of flank pain, hematuria, and decreased urine output.

PMID: 30725656
NOTE-- Diagnosis is with magnetic resonance angiography, Doppler ultrasonography, or computed tomography angiography.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
